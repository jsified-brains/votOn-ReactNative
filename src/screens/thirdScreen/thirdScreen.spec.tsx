import React from 'react';
import ThirdScreen from './thirdScreen';
import renderer from 'react-test-renderer';

describe('ThirdScreen component.', () => {
  it('matches expected snapshot', () => {
    const header = renderer.create(
      <ThirdScreen />
    ).toJSON;
    expect(header).toMatchSnapshot();
  });

  it('renders without crashing', () => {
    const rendered = renderer.create(<ThirdScreen />).toJSON();
    expect(rendered).toBeTruthy();
  });
});
