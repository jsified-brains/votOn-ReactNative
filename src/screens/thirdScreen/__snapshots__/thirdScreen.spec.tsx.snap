// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThirdScreen component matches expected snapshot 1`] = `[Function]`;
